### ISSUE TITLE
ValueError: Precision not allowed in integer format specifier

### ISSUE THREAD

**Issue by earino** [rhiever/tpot#235]
After running tpot for > 6 hours in my tpot_reprex example, the code crashed.

## Context of the issue

I was running the code I submitted yesterday as an issue, and received the following error message:

```
Best pipeline: MultinomialNB(DecisionTreeClassifier(input_matrix), 0.47000000000000003)
Traceback (most recent call last):
  File "example.py", line 36, in <module>
    tpot.fit(X_train, y_train)
  File "/usr/local/lib/python2.7/dist-packages/tpot/tpot.py", line 285, in fit
    halloffame=self.hof, verbose=False)
  File "/usr/local/lib/python2.7/dist-packages/deap/algorithms.py", line 162, in eaSimple
    offspring = toolbox.select(population, len(population))
  File "/usr/local/lib/python2.7/dist-packages/tpot/decorators.py", line 53, in wrapped_func
    format(self.gp_generation, gen_count_n_cols, high_score))
ValueError: Precision not allowed in integer format specifier
```

The version of the code which ran is available [here](https://app.dominodatalab.com/u/earino/tpot_reprex/view/example.py?commitId=fddfdc2fd22a2fd740dda2a035485fb95316d972), the run which actually generated the error can be seen [here](https://app.dominodatalab.com/u/earino/tpot_reprex/runs/57bb99eb80f4fe61775a16fa). The following is the relevant snippet of code:

```
X_train = train.ix[:, df.columns != "class"].as_matrix()
y_train = train["class"].as_matrix()
X_test = test.ix[:, df.columns != "class"].as_matrix()
y_test = test["class"].as_matrix()
 
#print X_train.shape, X_test.shape, y_train.shape, y_test.shape
 
tpot = TPOT(generations=5, verbosity=2)
tpot.fit(X_train, y_train)
print(tpot.score(X_test, y_test))
```

## Process to reproduce the issue

Using the following versions: deap-1.0.2 tpot-0.5.0 tqdm-4.8.4 update-checker-0.12

1. Execute the code (note that I have verbosity set? could this be a verbosity error)
2. Wait 6 hours
3. Error appears

## Expected result

A fitted ML pipeline

## Current result

The system crashes giving a `ValueError: Precision not allowed in integer format specifier` error.

## Possible fix

Unknown

**Comment by rhiever** [comment]
Looks like another issue popping up from Python 2.7. :-)

@teaearlgraycold, we should roll out a quick patch for this by removing the formatting. I have a related patch to that section of the code coming up, so I will include this in the patch.

**Comment by rhiever** [comment]
Hi @earino, try giving the TPOT v0.5.1 release a try (just now uploaded to PyPi) and please let us know if that fixed this issue. You might want to try it with a shorter TPOT run just to make sure it runs to completion and outputs everything as expected.

**Comment by earino** [comment]
Greetings @rhiever! 

Issue is fixed in v0.5.1:

https://app.dominodatalab.com/u/earino/tpot_reprex/runs/57bcdfed489465196851cb58

With the following packages installed:

* tpot-0.5.1 
* deap-1.0.2 
* tqdm-4.8.4 
* update-checker-0.12

Thanks for being so responsive! Now to actually get some TPOTing!
